src/main/java/com.lelek.springBoot/MySimpleMailMessage.java

```diff
@@ -0,0 +1,54 @@
+package com.lelek.springBoot;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+@Slf4j
+@Getter
+@Setter
+@Component
```

Comment:
If you're using `@Component` annotation without specifying scope, default one will be singleton. But this class has state which won't play well in multithreaded environment: imaging multiple threads will try to update sent property; what will be the outcome? In general, don't make stateful classes as spring components. This can still be created via new operator on demand. Consolidate persistence/storage logic inside one component: save from this class and read all from Sender should be in one place, so you know where to look if you want to understand how messages are stored